validator/node/node.go

```diff
@@ -104,6 +104,9 @@ func NewValidatorClient(cliCtx *cli.Context) (*ValidatorClient, error) {
 	if err := ValidatorClient.initializeFromCLI(cliCtx); err != nil {
 		return nil, err
 	}
+	//if err := ValidatorClient.db.UpdateProposalsProtectionDb(cliCtx.Context); err != nil {
+	//	return nil, err
+	//}
```

Comment:
Remove?

Reply:
was trying to find what causes the e2e failure

Reply:
removed the comment